app-i18n/translate-shell/Manifest

```diff
@@ -1 +1 @@
-DIST translate-shell-0.9.6.4.tar.gz 69788 SHA256 5ac8fd1a6b772750942f92665966ddf2b825594eb969770aabf16d7a83b3e6b9 SHA512 98c4a329794d38a5d22d98bf7bb821f1adc3e42dac96d2fd791e7cf64d785458d0f6006e83b21497f259525b1a70f306ab69090d3a12996610f742323a371147 WHIRLPOOL 53a9561738356d3ff101bd11ef96792d5cba5419a6db525a467fc3ff924ce97174f337ff8b7359ed3b58466280cf712d043973d48310c21c91e9914f934d7f08
+DIST translate-shell-0.9.6.6.tar.gz 73189 BLAKE2B ad4de403f2c95a1e4de5c963cb017f5a744f08d3bb8256dcc22f28c0916f6a0e693bce171a5dc500de5353c954bab7681f1227df9e48be286ad5aa07c5d06fa4 SHA512 c5451e537f44e325ee2f59d54942f91b8e2ccc0e322c42a2c4a0e98b8a933fe0630798a45e6f587ad5aee2cbb092204710ae36779c94b8333376a8072678d868
```

Comment:
Ok, please squash this with the earlier commit, i.e. do not introduce-then-remove intermediate versions.